README.md

```diff
@@ -1,2 +1,69 @@
-# opseng-challenge
-Wave OpsEng Development Challenge
+# Wave OpsEng Development Challenge
+
+Applicants for the [Operations Engineer career](https://www.waveapps.com/about-us/jobs/operations-engineer/) at Wave must complete the following challenge, and submit a solution prior to the interviewing process. This will help the interviewers assess your strengths, and frame the conversation through the interview process. Take as much time as you need, however we ask that you not spend more than a few hours.
```

Comment:
Ok. Hard to say, it was spread out over a few days.. I'd say probably 4-6 hours.